pom.xml

```diff
@@ -49,6 +49,11 @@
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>display-url-api</artifactId>
+            <version>1.0-SNAPSHOT</version>
```

Comment:
Unfortunately I am waiting on the hosting release https://issues.jenkins-ci.org/browse/HOSTING-179

The best way to build this PR would be to clone https://github.com/i386/display-url-api-plugin locally and build it.